### PR TITLE
Add Dmm Get Connections VIs

### PR DESCRIPTION
**Pull request by gpachecoNI**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds poly-VI implementation of Get Connections for the NI-DMM

### Why should this Pull Request be merged?

[AB#2533845](https://dev.azure.com/ni/DevCentral/_workitems/edit/2533845/) Create NI-DMM Specific Implementation of "Create Sessions.vi"

### What testing has been done?

Add Unit states based on DC-Power

### CODE REVIEW

File: Source/Tests/Tests.Runtime/Measurement Server/Get Connections/OneSessTwoPinOneInstrOneSiteReservation_CallNiDcPowerGetConnections1Pin1Connection_ReturnsOneConnection.vi
Comment (WesleyTangNationalInstruments) [inline Source/Tests/Tests.Runtime/Measurement Server/Get Connections/OneSessTwoPinOneInstrOneSiteReservation_CallNiDcPowerGetConnections1Pin1Connection_ReturnsOneConnection.vi:1]: What changes were made to this DCPower test?

File: Source/Tests/Tests.Runtime/Measurement Server/Get Connections/Test Inputs/One Session Two Pins One Instrument Two Sites - DMM.vi
Comment (WesleyTangNationalInstruments) [inline Source/Tests/Tests.Runtime/Measurement Server/Get Connections/Test Inputs/One Session Two Pins One Instrument Two Sites - DMM.vi:1]: Can DMMs have multiple channels per device? All of the DMMs I've seen have one channel per device (e.g. PXI-4070, PXI4072), but I'm not really familiar with DMMs.

File: Source/Tests/Tests.Runtime/Measurement Server/Get Connections/OneSessTwoPinOneInstrOneSiteReservation_CallNiDcPowerGetConnections1Pin1Connection_ReturnsOneConnection.vi
Comment (gpachecoNI) [reply Source/Tests/Tests.Runtime/Measurement Server/Get Connections/OneSessTwoPinOneInstrOneSiteReservation_CallNiDcPowerGetConnections1Pin1Connection_ReturnsOneConnection.vi:1]: Will revert.. probably just open and close it.. move a cable or something.  Miss to add the pics will upload in a sec.

File: Source/Tests/Tests.Runtime/Measurement Server/Get Connections/Test Inputs/One Session Two Pins One Instrument Two Sites - DMM.vi
Comment (gpachecoNI) [reply Source/Tests/Tests.Runtime/Measurement Server/Get Connections/Test Inputs/One Session Two Pins One Instrument Two Sites - DMM.vi:1]: mmm I guess I did not even look into it... Guess same issue with the CreateSessions test I was about to submit.

**Comment by gpachecoNI** [comment]
MeasurementServerTests_lvlib_ThreeSessTwoPinThreeInstrTwoSiteReservation__CallNiDmmGetConnectionsNPinsMConnections__ReturnsThreeConnectionsd
MeasurementServerTests_lvlib_ThreeSessTwoPinThreeInstrTwoSiteReservation__CallNiDmmGetConnections1PinMConnections__ReturnsTwoConnectionsd
MeasurementServerTests_lvlib_ThreeSessTwoPinThreeInstrTwoSiteReservation__CallNiDmmGetConnections1Pin1Connection__ReturnsOneConnectiond
MeasurementServerTests_lvlib_Three_Session_Two_Pins_Three_Instrument_Two_Sites_-_DMMd

File: Source/Runtime/MeasurementLink niDMM/Get Connections - 1Pin 1Conn.vi
Comment (pbirkhol-ni) [inline Source/Runtime/MeasurementLink niDMM/Get Connections - 1Pin 1Conn.vi:1]: The help for this VI (and the other poly instances) mentions the old name for the VI. You should update it to use the new name.

```This VI returns an error if multiple connections match the inputs. Use <b>niDMM Get Connections - 1PinMConnection.vi</b> to get multiple connections from a single pin.```

Comment (pbirkhol-ni) [reply Source/Runtime/MeasurementLink niDMM/Get Connections - 1Pin 1Conn.vi:1]: This is pretty minor, but this VI is saved with the error case visible. You should save it with the most useful case visible, which is often the non error case.

**Review by pbirkhol-ni — APPROVED** [review]

**Pull request merged**